bundle/mvn/camunda-saas-bundle/src/test/java/io/camunda/connector/runtime/TestSpringContextStartup.java

```diff
@@ -22,8 +22,13 @@
 
 @SpringBootTest(
     classes = SaaSConnectorRuntimeApplication.class,
-    properties = {"camunda.saas.secrets.projectId=42", "camunda.connector.polling.enabled=false"})
-@ActiveProfiles("dev")
+    properties = {
+      "camunda.saas.secrets.projectId=42",
+      "camunda.connector.polling.enabled=false",
+      "operate.client.enabled=false",
```

Comment:
Looks like this property is also not needed here (already applied in the main config file) 🙂

Reply:
Removed it. 👍